content/en/agent/guide/autodiscovery-with-jmx.md

```diff
@@ -175,62 +306,94 @@ apiVersion: v1
 kind: Pod
 # (...)
 metadata:
-  name: '<POD_NAME>'
-  annotations:
-    ad.datadoghq.com/<CONTAINER_IDENTIFIER>.check.id: 'CUSTOM_AD_IDENTIFIER'
-    # (...)
+    name: '<POD_NAME>'
+    annotations:
+        ad.datadoghq.com/<CONTAINER_IDENTIFIER>.check.id: '<CUSTOM_AD_IDENTIFIER>'
+        # (...)
 spec:
-  containers:
-    - name: '<CONTAINER_IDENTIFIER>'
+    containers:
+        - name: '<CONTAINER_IDENTIFIER>'
+          # (...)
+          env:
+            - name: POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
+
+            - name: JAVA_OPTS
+              value: >-
+                -Xms256m -Xmx6144m
+                -Dcom.sun.management.jmxremote
+                -Dcom.sun.management.jmxremote.authenticate=false
+                -Dcom.sun.management.jmxremote.ssl=false
+                -Dcom.sun.management.jmxremote.local.only=false
+                -Dcom.sun.management.jmxremote.port=<JMX_PORT>
+                -Dcom.sun.management.jmxremote.rmi.port=<JMX_PORT>
+                -Djava.rmi.server.hostname=$(POD_IP)
 # (...)
 ```
 
 **Note**:
 
-* To apply a specific configuration to a given container, Autodiscovery identifies containers by **name**, _not_ by image. It tries to match `<CONTAINER_IDENTIFIER>` to `.spec.containers[0].name`, not `.spec.containers[0].image`
-* If you define your Kubernetes pods directly with `kind: Pod`, add each pod's annotations directly under its `metadata` section. If you define pods indirectly with replication controllers, ReplicaSets, or deployments, add pod annotations under `.spec.template.metadata`.
+- To apply a specific configuration to a given container, Autodiscovery identifies containers by **name**, _not_ by image. It tries to match `<CONTAINER_IDENTIFIER>` to `.spec.containers[0].name`, not `.spec.containers[0].image`
+- If you define your Kubernetes pods directly with `kind: Pod`, add each pod's annotations directly under its `metadata` section. If you define pods indirectly with replication controllers, ReplicaSets, or deployments, add pod annotations under `.spec.template.metadata`.
+- The `JAVA_OPTS` environement variable needs to be created, so that your JMX server allows the agent to connect to the RMI registry.
```

Comment:
```suggestion
- The `JAVA_OPTS` environment variable needs to be created, so that your JMX server allows the agent to connect to the RMI registry.
```